targets/XPlatCppSdk/source/build/iOS/TestIOSApp/TestAppIOS.mm

```diff
@@ -11,8 +11,7 @@
 #define USE_EXTERNAL_JSON_FILE
 
 #ifdef USE_EXTERNAL_JSON_FILE
-#warning "Replace below file name(except ext) with yours, and DO NOT SHARE IT."
-static constexpr NSString* c_jsonFileName = @"YOUR_JSON_FILE_NAME";
+#warning "Replace the contents of $WORKSPACE/TestTitleData/testTitleData.json with your title secret information!"
```

Comment:
$workspace isn't a variable for anybody outside of Jenkins.

Reply:
eh, i don't think that's an issue.  it's just a comment and "workspace" is commonly used to mean the project directory.  i think they'll get the picture.